sklearn/decomposition/factor_analysis.py

```diff
@@ -108,6 +108,20 @@ class FactorAnalysis(BaseEstimator, TransformerMixin):
     n_iter_ : int
         Number of iterations run.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_digits
+    >>> from sklearn.decomposition import FactorAnalysis
+    >>> X, _ = load_digits(return_X_y=True)
+    >>> transformer = FactorAnalysis(n_components=10, random_state=0)
+    >>> transformer.fit(X)
+    FactorAnalysis(copy=True, iterated_power=3, max_iter=1000, n_components=10,
+            noise_variance_init=None, random_state=0, svd_method='randomized',
+            tol=0.01)
```

Comment:
Open question (also to other reviewers): what's the added value of systematically checking the signature here? This would need to be edited each time parameters are changed/added etc. Would calling `fit_transform` directly be much less clear?